modules/mixins/Helpers.js

```diff
@@ -98,14 +98,14 @@ var Helpers = {
           }).bind(this));
 
           scrollSpy.addSpyHandler((function(y) {
-
+            
             if(!element) {
                 element = scroller.get(to);
-
-                var cords = element.getBoundingClientRect();
-                elemTopBound = (cords.top + y);
-                elemBottomBound = elemTopBound + cords.height;
             }
+            var cords = element.getBoundingClientRect();
```

Comment:
We still have "if" statement in that case:
`if(!element) {
    element = scroller.get(to);
}`